robot.rb

```diff
@@ -1,19 +1,22 @@
 # frozen_string_literal: true
 
+ROBOT_VIEW = { NORTH: '>', SOUTH: '<', EAST: 'v', WEST: '^' }.freeze
+DIRECTIONS = %w[WEST NORTH EAST SOUTH].freeze
+STEP_INFO = { NORTH: { x: 0, y: 1 }, SOUTH: { x: 0, y: -1 }, EAST: { x: 1, y: 0 }, WEST: { x: -1, y: 0 } }.freeze
```

Comment:
I assume that the initial task was a simple movement so it is just a recommendation. 
This approach is relevant for the rectangular movement with the rectangular cells with the size multiple 1.
It will not be much for the cases with step 1.5 or rotation 10 degrees. It is better to be one step ahead and as additional questions about requirements for the expansion of the requirements. In real life you need to be ready for this.